src/commands/build-util.ts

```diff
@@ -49,9 +49,10 @@ export async function buildFile(
         const destBuildFile = output[destExt];
         const result = await step.transform({
           contents: destBuildFile,
+          extension: destExt,
```

Comment:
Minor change 1: we added a new `extension` param to the `build()` option. This is completely a convenience, as this could be derived from `filePath`, but it would help a lot of plugins from having to do `path.extname()` over and over again.

Reply:
sounds good, but I'd prefer `fileExt` to match `filePath`